src/main/scala/com/databricks/labs/overwatch/utils/PipelineValidationHelper.scala

```diff
@@ -0,0 +1,684 @@
+package com.databricks.labs.overwatch.utils
+
+import com.databricks.labs.overwatch.pipeline.Pipeline
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.sql.functions._
+import com.databricks.labs.overwatch.pipeline._
+import com.databricks.labs.overwatch.pipeline.TransformFunctions._
+import com.databricks.labs.validation._
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.expressions.Window
+
+import java.time.LocalDateTime
+import scala.collection.mutable.ArrayBuffer
+
```

Comment:
why abstract class?
Please make documentation for the class

Reply:
@sriram251-code  Have changed it to Normal Class with Documentation.